config/shortcutsConfig.json

```diff
@@ -165,6 +165,10 @@
 		"mainMenuCampaignCustom":   "C",
 		"mainMenuCampaignRoe":      "R",
 		"mainMenuCampaignSod":      "S",
+		"mainMenuCampaignChr":      "K",
```

Comment:
Perhaps 'T' for Tarnum? Not very obvious, but neither is 'K'

Reply:
"T" would be fine aswell, will change it.

Reply:
Changed Chronicles shortcut to "T"